docs/src/main/paradox/java/http/server-side/server-https-support.md

```diff
@@ -0,0 +1 @@
+../../../scala/http/server-side/server-https-support.md
```

Comment:
II would even suggest to drop the `server-` and `client-` prefixes from the name on both the server-side and client-side pages.